Expensify/App
# Add more logs to try to resolve "cb() not called" error

Please review @Jag96 

### Details
1. Updated to use correct node on iOS (Node 14.x)
2. Turned on "silly" debug mode, which logs the entire kitchen sink to hopefully spot the cause of the "cb error"

### Fixed Issues
Related https://github.com/Expensify/Expensify/issues/152345

### Tests
1. Run `npm install`
2. Verify it installs correctly 👍 

### Tested On
